service_info_cms/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from cms.extensions import PageExtensionAdmin
+
+from .models import IconNameExtension
+
+
+class IconNameExtensionAdmin(PageExtensionAdmin):
```

Comment:
I can view http://localhost:4005/en/admin/service_info_cms/iconnameextension/ but I can't view http://localhost:4005/en/admin/service_info_cms/ 

How does that happen?